### PR TITLE
feat: Adding Speech Recognition notebook using Chirp

### PR DESCRIPTION
**Pull request by inardini**
This notebook demonstrates how to use Chirp for accurately converting voice to text.

### CODE REVIEW

File: speech/getting-started/speech_recognition.ipynb [hunk 1]
```diff
@@ -0,0 +1,915 @@
+{
```
Comment (erwinh85) [inline speech/getting-started/speech_recognition.ipynb:1]: <p>There is no way to use the vertex sdk?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (erwinh85) [inline speech/getting-started/speech_recognition.ipynb:1]: <p>Can we link to the audio file / license?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (erwinh85) [inline speech/getting-started/speech_recognition.ipynb:1]: <p>Could use a bit more explanation. Why short audio first? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (erwinh85) [inline speech/getting-started/speech_recognition.ipynb:1]: <p>Same here. Could use a bit more explanation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (erwinh85) [inline speech/getting-started/speech_recognition.ipynb:1]: <p>Do we want to link to documentation on what audio formats can be used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (erwinh85) [inline speech/getting-started/speech_recognition.ipynb:1]: <p>What's a recognizer?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (erwinh85) [inline speech/getting-started/speech_recognition.ipynb:1]: <p>I don't thing this formatting is correct?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

**Comment by polong-lin** [comment]
Could you also update the table in the main README.md with a link to this?

File: speech/getting-started/speech_recognition.ipynb (hunk 1 above)
Comment (polong-lin) [inline speech/getting-started/speech_recognition.ipynb:1]: <p>Please add link to documentation. I would also suggest removing the research paper as it's not necessary and is at best distracting to the developer experience in this notebook.</p><p> </p><p>I would also add a little bit more about what's possible with Chirp and why developers should consider using Chirp. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (polong-lin) [inline speech/getting-started/speech_recognition.ipynb:1]: <p>Please remove bullet point that is residual from the template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (inardini) [reply speech/getting-started/speech_recognition.ipynb:1]: <p>done</p>

Comment (inardini) [reply speech/getting-started/speech_recognition.ipynb:1]: <p>done</p>

Comment (inardini) [reply speech/getting-started/speech_recognition.ipynb:1]: <p>done</p>

Comment (inardini) [reply speech/getting-started/speech_recognition.ipynb:1]: <p>It is just a matter of order short-long</p>

Comment (inardini) [reply speech/getting-started/speech_recognition.ipynb:1]: <p>Can you be more specific?</p>

**Review by lavinigam-gcp — APPROVED** [review]
LGTM

**Pull request merged**